backend/spec/workers/archive_hubee_portail_enrollments_worker_spec.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe ArchiveHubeePortailEnrollmentsWorker, type: :worker do
+  subject { described_class.new }
```

Comment:
As we do have one method to check here (if we do send email with a mailing list independently from this PR)

`subject { described_class.new.perform } `